### DISCUSSION TITLE
Next.js Slack Workspace

### DISCUSSION THREAD

**Discussion by enkicoma** [vercel/next.js#16887] [Help] [answered]
Is there a next.js slack community?

**Comment by stfnsr** [comment]
If I remember correctly, next had a section on spectrum.chat, now there’s GitHub discussions instead.
Alternatively though, there’s a nextjs channel on reactiflux: https://www.reactiflux.com/

**Comment by Seybo** [comment]
i think it's a good idea to create a slack space. I'm quite surprised actually it's still not there

**Comment by mayankzalavadia** [comment]
I also think they have to create a slack channel for cumunity discussion.

**Comment by miguderp** [comment, marked as answer]
There's an official Discord channel: https://discord.com/invite/w8p9Psrk2M
I think it launched at the same time as their NextJS conf'

**Reply by mayankzalavadia** [reply]
@miguderp Thank you for sharing this link.

I also need one more help.

Can you please, suggest to me a proper Nginx configuration for the nextjs. Which will cover all types of speed optimization settings.

Thank you in advance!

**Comment by ndungtse** [comment]
> Is there a next.js slack community?

Currently I can't join the server with this link. Does the server still accepts new members?

**Comment by tomhayes** [comment]
> > Is there a next.js slack community?
> 
> Currently I can't join the server with this link. Does the server still accepts new members?

Link is now https://discord.com/invite/bUG2bvbtHy

**Reply by ndungtse** [reply]
Thanks, this works perfectly

**Comment by jasonching95** [comment]
Please let me join in the channel too.
Thanks
